metis/lib/models/file.rb

```diff
@@ -111,7 +111,7 @@ def self.link_to_block(params)
       dest_folder_path, dest_file_name = Metis::File.path_parts(params[:dest_file_path])
 
       dest_bucket = Metis::Bucket.find(
-        project_name: params[:project_name],
+        project_name: params[:dest_project_name] || params[:project_name],
```

Comment:
Does this elegantly handle invalid project_names?  We get a long of rollbar errr pings for simple invalid parameter issues, if we could have a non 500 response to bad dest project name, that would be ideal.

Reply:
That should get handled at the controller level already, in the sense that the project is compared to the user's list of permissions. If the user doesn't have access to either of these projects (or the project doesn't exist), the controller returns a 422 about an invalid bucket and the action should never reach this point.

We could check again here against the user's permissions, but it feels redundant.